sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSensorTest.java

```diff
@@ -125,20 +125,6 @@ public void noAnalysisIfNoFilesDetected() throws Exception {
   }
 
   @Test
-  public void failWhenOsIsNotWindows() throws MessageException {
-    // Arrange
-    addFileToFs();
-    when(system.isOsWindows()).thenReturn(false);
-
-    // Assert exception
-    thrown.expect(MessageException.class);
-    thrown.expectMessage("C# analysis is not supported");
-
-    // Act
-    sensor.execute(tester);
-  }
-
-  @Test
```

Comment:
Something's weird here. Either you have removed an attribute you shouldn't have or you forgot to remove a method.

Reply:
Forget about it....I haven't seen top `@Test` wasn't removed.